pyproject.toml

```diff
@@ -64,5 +64,7 @@ markers = [
 log_format = "%(asctime)s.%(msecs)03d %(levelname)s %(message)s"
 
 [tool.coverage.run]
+concurrency = ["multiprocessing"]
```

Comment:
Needed by pytest-cov, see https://pytest-cov.readthedocs.io/en/latest/subprocess-support.html#if-you-use-multiprocessing for more information

Reply:
I would add this as a comment in the file